BaseStation/BaseStation/BaseStation.csproj

```diff
@@ -5,7 +5,7 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{717939F0-9BDC-4D66-99B5-AE34D586488E}</ProjectGuid>
-    <OutputType>WinExe</OutputType>
+    <OutputType>Exe</OutputType>
```

Comment:
You can't just convert this to a console application! We still need the UI to spawn in which is only does as a Windows Application. You'll have to spawn the command prompt as a child process and redirect standard out to it. If you leave it as a Windows Application and have no command process it will write to Visual Studio console, but we can't assume Visual Studio will always be running.

Reply:
It wasn't running on my computer when I tested it. I'll try again tonight!